GameOfLife/GameOfLife/Conway/ConwayBoard.cs

```diff
@@ -0,0 +1,150 @@
+﻿
+using GameOfLife.Utils;
+using System.Runtime.Serialization;
+
+namespace GameOfLife
+{
+    /// <summary>
+    /// Board for Conway's game of life.
+    /// </summary>
+    [DataContract]
+    public class ConwayBoard : Board
+    {
+        /// <summary>
+        /// Count of live cells on the board.
+        /// </summary>
+        [DataMember]
+        private int _liveCellCount { get; set; } = 0;
+
+        /// <summary>
+        /// Count of iterations of the board.
+        /// </summary>
+        [DataMember]
+        private int _iterationCount { get; set; } = 1;
+
+        /// <summary>
+        /// Initialiazes board with required size.
+        /// Fills board with random boolean values.
+        /// </summary>
+        /// <param name="height">Height of the board.</param>
+        /// <param name="width">Width of the board.</param>
+        public ConwayBoard(int height, int width) : base(height, width)
+        {
+            Random randomGen = new Random();
+
+            for (int i = 0; i < height; i++)
+            {
+                for (int j = 0; j < width; j++)
+                {
+                    InitialBoard[i, j] = randomGen.Next(0, 2) == 1;
+
+                    _liveCellCount += InitialBoard[i, j] ? 1 : 0;
+                }
+            }
+        }
+
+        /// <summary>
+        /// Defines algorithm of how game should be played and displayed.
+        /// </summary>
+        public override void Flow()
+        {
+            DisplayBoard();
+
+            Panel.DisplayStatsTableRow(Labels.GenOfCells, _iterationCount);
+            Panel.DisplayStatsTableRow(Labels.AmountOfLiveCells, _liveCellCount);
+
+            Iterate();
+
+            Thread.Sleep(1000);
+
```

Comment:
Please remove empty line, and in other places as well.

Reply:
Done!